Jenkinsfile-2

```diff
@@ -0,0 +1,59 @@
+def make_command(String, folder) {
```

Comment:
Why not just replace the existing jenkinsfile? It'll pick it up and run it in the branch run (but not the PRs)

Reply:
I probably will if I get it right, if not, I'll throw it away. 

I've no idea what the tag stag is actually doing.